docker/Dockerfile

```diff
@@ -14,7 +14,7 @@
 
 ARG BASEIMAGE=registry.k8s.io/build-image/debian-base:bookworm-v1.0.0
 
-FROM golang:1.21@sha256:337543447173c2238c78d4851456760dcc57c1dfa8c3bcd94cbee8b0f7b32ad0 as builder
+FROM golang:1.21@sha256:2ff79bcdaff74368a9fdcb06f6599e54a71caf520fd2357a55feddd504bcaffb as builder
```

Comment:
The sha for latest `1.21.5` image is [`135ba6c2ae2a9a022f53f3eee24ef213bc62e876811cd9582b91045168206873`]( https://hub.docker.com/layers/library/golang/1.21/images/sha256-135ba6c2ae2a9a022f53f3eee24ef213bc62e876811cd9582b91045168206873?context=explore)